### PR TITLE
Fix release too many args issue

### PR DESCRIPTION
**Pull request by MrWWheat**

### CODE REVIEW

File: src/fuse/mod.rs [hunk 1]
```diff
@@ -151,6 +151,16 @@ pub struct FsSetxattrParam<'a> {
     pub position: u32,
 }
 
+/// Param passed to setxattr
+#[derive(Debug)]
+pub struct FsReleaseParam {
```
Comment (pzheng1025) [inline src/fuse/mod.rs:156]: Add document for each field. 
Refer to https://github.com/datenlord/sync_fuse/commit/6f68fa406c644c1d7473f296ccd0b94e5f070722

**Review by pzheng1025 — CHANGES_REQUESTED** [review]

File: src/fuse/mod.rs (hunk 1 above)
Comment (MrWWheat) [reply src/fuse/mod.rs:156]: Done.

File: src/fuse/request.rs [hunk 2]
```diff
@@ -17,7 +17,10 @@ use super::channel::FuseChannelSender;
 use super::ll_request;
 use super::reply::{Reply, ReplyDirectory, ReplyEmpty, ReplyRaw};
 use super::session::{Session, BUFFER_SIZE, MAX_WRITE_SIZE};
-use super::{Cast, Filesystem, FsSetattrParam, FsSetxattrParam, FsWriteParam};
+use super::{
+    Cast, Filesystem, FsExchangeParam, FsGetlkParam, FsReleaseParam, FsSetattrParam, FsSetlkParam,
```
Comment (pzheng1025) [inline src/fuse/request.rs]: Please fix these unused import issue.
Looks like only Macos use them.

**Review by pzheng1025 — CHANGES_REQUESTED** [review]

File: src/fuse/request.rs (hunk 2 above)
Comment (MrWWheat) [reply src/fuse/request.rs]: Done.

**Pull request closed without merging**